src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx

```diff
@@ -0,0 +1,33 @@
+import { LeftOutlined, RightOutlined } from '@ant-design/icons'
+import { Point } from 'src/pages/timeBasedMap'
+import './MapFooterButtons.scss'
+
+type TMapFooterButtons = {
+  index: number
+  positions: Point[]
+  navigateMarkers: (id: number) => void
+}
+
+function MapFooterButtons({ index, positions, navigateMarkers }: TMapFooterButtons) {
+  const rightStep = index + 1
+  const leftStep = index - 1
+
+  const checkIfValidStep = (i: number) => {
+    return Boolean(positions.at(i))
+  }
+
+  return (
+    <div className="map-footer-buttons">
+      <RightOutlined
+        className={`${checkIfValidStep(rightStep) ? '' : 'disabled'}`}
+        onClick={() => navigateMarkers(rightStep)}
+      />
+      <LeftOutlined
```

Comment:
we can add a "title" attribute for hover ("prev" / "next")

Reply:
I choose to use right/left-chevron instead cuz  'prev' 'next' depend on the direction 
<img width="926" alt="image" src="https://github.com/hasadna/open-bus-map-search/assets/20385035/c64a02f6-cf15-43f8-b21f-6a66808eb63a">

Reply:
the title attribute is visible to users, and should help them understand whether right will indicate the previous or the next item of the list.
In RTL website, it's common to see right arrows serving as "next" buttons, while on LTR website it's the opposite, that's why I feel having this title can be significant to the user experience
 
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/a3508d3d-9613-4a39-b75d-8de3bfa26cb7)